info.md.erb

```diff
@@ -0,0 +1,18 @@
+<%
+  jupyterlab_version = JSON.parse(user_defined_context_file.read)['version'].split('/').last
+  jupyterlab_switch = JSON.parse(user_defined_context_file.read)['jupyterlab_switch']
+  jupyterlab_use_extension = JSON.parse(user_defined_context_file.read)['jupyterlab_use_extension']
+  jupyterlab_extension_enabled = false
+  if jupyterlab_switch == "1" and jupyterlab_use_extension == "1" and jupyterlab_version.split('.').first.to_i > 0
+    jupyterlab_extension_enabled = true
+  end
+%>
```

Comment:
```ruby
begin
  data = JSON.parse(user_defined_context_file.read)
rescue
  data = {}
end

version = data['version'].to_s.split('/').last 
# the the other stuff. casting to string above `to_s` ensures it's safe
```

Reply:
No problem. It is done in d9c96f47c43915c85ed85ea5868d00dd9467cc37.